### PR TITLE
Improve output of failed tests

### LINKED ISSUE #235: Improve feedback of failed assertions in LUA
Messages from failed assertions are presented with `<LOG::>` instead of being a part of `<FAILED::>` :

image

Can this be improved, or maybe such behavior is caused by some quirks of LUA testing framework?

The issue seems to be similar to #222 but I am not sure it has the same cause.

### PR DESCRIPTION
**Pull request by hobovsky**
Fixes codewars/runner#235.

Changed two types of output events:
- Failing assertions
- Errors outside of `it` block

With new reporter:

image

Original reporter:

image

No reporter:

image

### CODE REVIEW

File: src/busted/outputHandlers/codewars.lua [hunk 1]
```diff
@@ -29,9 +29,9 @@ return function(options)
   end
 
   local function onTestFailure(element, parent, message, debug)
-    print('\n<FAILED::>Test Failed')
     -- remove '/home/codewarrior/lua/fixture.lua:\d+: ' from message
-    print('\n<LOG:ESC:>' .. escape(message:sub(message:find(' ') + 1)))
+    local msg = escape(message:sub(message:find(' ') + 1))
+    print('\n<FAILED::>Test Failed<:LF:>' .. msg)
```
Comment (kazk) [inline src/busted/outputHandlers/codewars.lua]: Maybe it's better without `Test Failed`?

Comment (hobovsky) [reply src/busted/outputHandlers/codewars.lua]: Yup, I like it more with just the assertion message, but I was wondering if there's any convention among all reporters but I did not check.

If you dont have any specific practice across all reporters, i think just the message is good.

Comment (hobovsky) [reply src/busted/outputHandlers/codewars.lua]: Now it is:

image

I think it's nicer this way.

**Review by kazk — APPROVED** [review]

**Comment by hobovsky** [comment]
Added support for newlines in titles:

image

**Pull request merged**